src/components/theme/index.ts

```diff
@@ -5,3 +5,4 @@ export * from './useTheme';
 export * from './useThemeValue';
 export * from './withTheme';
 export * from './withThemeValue';
+export {Theme as ThemeType} from './types';
```

Comment:
Why alias?

Reply:
```js
import {Theme} from '@yandex-cloud/uikit'
```

I think it looks like we import some component

Reply:
We already have similar exported types (i.e. `Lang`). No need to add *Type

Reply:
Fixed